src/nih_wayfinding/app/scripts/app.js

```diff
@@ -2,7 +2,14 @@
     'use strict';
 
     /* ngInject */
-    function DefaultRoutingConfig($urlRouterProvider) {
+    function DefaultRoutingConfig($urlRouterProvider, localStorageServiceProvider) {
+
+        // set up local storage
+        localStorageServiceProvider
```

Comment:
I think it would be better if this configuration got moved to the nih.profiles module.js, since the app level doesn't care about localstorage, and the dependency is defined at the profile module level.